emcache/_cython/hashing/rendezvous.pxd

```diff
@@ -2,9 +2,7 @@
 # Copyright (c) 2020-2024 Pau Freixes
 
 cdef class RendezvousNode:
-    cdef:
```

Comment:
It seems I'm not entirely right here. it depends on readability and understanding. https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#grouping-multiple-c-declarations